tests/000-flaky/bugs_distribute_bug-1117851.t

```diff
@@ -76,16 +65,9 @@ TEST create_files $M0
 ## Mount FUSE with caching disabled (read-write) again
 TEST glusterfs --entry-timeout=0 --attribute-timeout=0 -s $H0 --volfile-id $V0 $M1;
 
-TEST move_files $M0
-TEST move_files $M1
-
-# It's regrettable that renaming 1000 files might take more than 30 seconds,
-# but on our test systems sometimes it does, so double the time from what we'd
-# use otherwise.  There still seem to be some spurious failures, 1 in 20 when
-# this does not complete, added an additional 60 seconds to take false reports
-# out of the system, during test runs, especially on slower test systems.
-EXPECT_WITHIN 120 "done" cat $M0/status_0
-EXPECT_WITHIN 120 "done" cat $M1/status_1
+move_files $M0 &
+move_files $M1 &
+wait
```

Comment:
How to validate the files are successfully renamed earlier we do check the same based on the string "done" .

Reply:
The same time when "done" was written is when `wait` will return. The "done" was written when all renames finished, independently of whether they failed or not.

In the hypothetical case that doing renames something hangs, the global timeout of the script will already terminate it. It's not necessary to use a fixed timeout that may trigger when everything is working fine but slower than normal for some reason.